tests/test_solr_prep.py

```diff
@@ -22,27 +22,33 @@ def test_legislative_session(bill, metro_organization, event, mocker, month):
 
     now = datetime.now()
 
+    # Prevent ValueError
+    if month == 6 and now.day == 31:
+        today = 30
+    else:
+        today = now.day
```

Comment:
While github ran tests, I got a ValueError here because the month being tested was 6 and `now.day` was the 31st. So this is just a failsafe.